lib/github.js

```diff
@@ -85,53 +86,89 @@ function aliasGet (msg, done) {
 }
 
 function queryGithub (msg, done) {
-  var cache = opts.cache
+  var cache  = opts.cache
 
   github.authenticate({
-    type: 'token',
+    type : 'token',
     token: opts.token
   })
 
-  var params = {
-    user: msg.user,
-    repo: msg.repo
-  }
-
-  github.repos.get(params, (err, repo) => {
-    if (err) return done(err)
-    if (!repo) return done()
+  async.parallel({
+    getRepository  : function (callbackFn) {
```

Comment:
npm run test npm run lint will ensure the code follow is the correct convention for you. It will give you each line were the code does not follow the linter convention.